src/utilities/date/FormatDate.js

```diff
@@ -0,0 +1,143 @@
+import moment from 'moment';
+
+const TOTAL_MONTH = 12;
+
+export const format = date => {
+  return moment(date).format('MMMM YYYY');
+};
+
+/**
+ * Convert month to year and month .
+ *
+ * @param {integer} monthCount
+ * @returns {Object}
+ */
+export const getYearMonth = monthCount => {
+  const m = monthCount % TOTAL_MONTH;
+  const y = Math.floor(monthCount / TOTAL_MONTH);
+
+  return { year: y, month: m };
+};
+
+export const getExperienceFormat = experience => {
+  if (experience.type === 'Year') {
+    return { year: Number(experience.value), month: 0 };
+  } else {
+    const result = getYearMonth(experience.value);
+
+    return { ...result };
+  }
+};
+
+/**
+ * Gett difference between two dates in years and months .
+ *
+ * @param {Date} startDate Start date.
+ * @param {Date} endDate  End date.
+ * @returns {Object} Difference.
+ */
+export const getDifferenceInYearMonth = (startDate, endDate) => {
+  const formattedEndDate = moment(endDate);
+  const formattedStartDate = moment(startDate);
+
+  const years = formattedEndDate.diff(formattedStartDate, 'year');
+
+  formattedStartDate.add(years, 'years');
+
+  const months = formattedEndDate.diff(formattedStartDate, 'months');
+
+  formattedStartDate.add(months, 'months');
+
+  return {
+    year: years,
+    month: months,
+  };
+};
+
+/**
+ * Get Label for duration in years and months.
+ *
+ * @param {Date} startDate Start date.
+ * @param {Date} endDate End date.
+ * @param {bool} currentlyWorking Status if currently engaged or not.
+ * @returns {string}
+ */
+export const getDifferenceYearMonth = (startDate, endDate, currentlyWorking) => {
+  let labelForDifference = '';
+
+  const diff = getDifferenceInYearMonth(startDate, currentlyWorking ? new Date() : endDate);
+
+  if (diff.year !== 0) {
+    labelForDifference = diff.year > 1 ? `${diff.year} years ` : `${diff.year} year `;
+    if (diff.month !== 0) {
+      labelForDifference += ' and ';
+    }
+  }
+
+  if (diff.month !== 0) {
+    labelForDifference += diff.month > 1 ? ` ${diff.month} months` : ` ${diff.month} month`;
```

Comment:
it can use the `timeWithSuffix` function for this too

Reply:
done